docs/content/en/docs/packages/packagecontroller.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Manging the package controller"
+linkTitle: "Package controller"
+weight: 4
+aliases:
+    /docs/tasks/packages/packagecontroller/
+---
+
+### Install package controller after installation
+
+{{% alert title="Important" color="warning" %}}
+**Recommendation**: Ideally, you should never delete package controller after installation. When the package bundle controller resource is deleted and then gets recreated, the active bundle recorded in the package bundle controller resource will point to the latest bundle, but you can always go back to the previous bundle version. See more information about managing package bundles [here]({{< relref "./packagebundles" >}})
+{{% /alert %}}
+
+
+If you created a cluster without the package controller or if the package controller was not properly configured, you may need to do some things to enable it.
+
+Make sure you are authenticated with the AWS CLI. Use the credentials you set up for packages. These credentials should have [limited capabilities]({{< relref "./prereq#setup-authentication-to-use-curated-packages" >}}):
+
+```bash
+export AWS_ACCESS_KEY_ID="your*access*id"
+export AWS_SECRET_ACCESS_KEY="your*secret*key"
+export EKSA_AWS_ACCESS_KEY_ID="your*access*id"
+export EKSA_AWS_SECRET_ACCESS_KEY="your*secret*key"
+```
+
+Verify your credentials are working:
+```shell
+aws sts get-caller-identity
+```
+Login to docker
+```bash
+aws ecr get-login-password |docker login --username AWS --password-stdin 783794618700.dkr.ecr.us-west-2.amazonaws.com
+```
+
+Verify you can pull an image
+```bash
+docker pull 783794618700.dkr.ecr.us-west-2.amazonaws.com/emissary-ingress/emissary:v3.5.1-bf70150bcdfe3a5383ec8ad9cd7eea801a0cb074
+```
+If the image downloads successfully, it worked!
+
+If you do not have the package controller installed (it is installed by default), install it now:
+```shell
+eksctl anywhere install packagecontroller -f cluster.yaml
+```
+If you had the package controller disabled, you may need to modify your `cluster.yaml` to enable it.
```

Comment:
and run eks-a cli upgrade?

Reply:
No, this is suppose to be a follow up if you had it disabled in your `cluster.yaml` it seems. I can move this further up in the steps to make this more clear.